### DISCUSSION TITLE
CHIME operation schedule

### DISCUSSION THREAD

**Discussion by yniino** [chime-frb-open-data/community#43] [Q&A] [answered]
Hi, 

I would like to know the operation schedule of CHIME (e.g., expected downtime) to organize observation of our observatory. Are you announcing the operation schedule on the email list (or somewhere else)? 

Thanks,
Yuu

**Comment by andrewzwaniga** [comment, marked as answer]
Hi Yuu,

Thanks for your question. We will provide advance notice of upcoming planned periods of significant downtime on a best-effort basis. For now we will communicate this through the Google Group that all email subscribers are part of. We will also post an [announcement](https://github.com/chime-frb-open-data/community/discussions/categories/announcements). In the future we hope to have such notices also visible on the CHIME/FRB public [webpage](https://www.chime-frb.ca/).

In general, uptime of the VOEvent Service is subject to change without notice. However, if there is an interruption we will do our best to communicate this through the Google Group mailing list, and/or through the announcements, as described above.

If you have any other questions, please do not hesitate to ask.

Best regards,
Andrew
